styleguide/_themes/derek/scss/snowflakes/stories.scss

```diff
@@ -0,0 +1,94 @@
+.story-colLeft {
+  @include make-md-column(6);
+}
+
+.story-colRight {
+  @include make-md-column(5);
+  @include make-md-column-push(1);
+}
+
+.story-title {
+  text-align: left;
+}
+
+.story-caseStudy-button {
+  @include button;
+  @include button-learning;
+  margin-bottom: 1rem;
+  width: auto;
```

Comment:
Yeah. so either we need to set it into a container or we do this until we change the widths on buttons entirely. Since this would be the only two places we're changing the way we set widths Im concerned with the impending change to buttons and ctas that this may end up wonky. @ejchiefs you want to weight in on this one?